Reference/Config/customErrorPages/index.md

```diff
@@ -0,0 +1,81 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to implement custom error pages
+
+Since Umbraco is built upon Microsoft's .NET Framework and is using ASP.NET, you have several options when it comes to setting up custom error pages on your website.
+
+## In-code error page handling
+
+One way is to watch for error events and serve corresponding pages via C# code. Please refer to the following page for an example - [Custom 404 handlers](../404handlers/).
+
+## Config file handling
+
+A quicker implementation - you can specify a page that will be displayed whenever the application encounters a particular error.
```

Comment:
What is "quick" for some, might be "slow" for others ;) 
Try to make a tutorial as neutral as possible.

```suggestion
Another approach for implementing a custom error page would be to specify a page that will be displayed whenever the application encounters a particular error.
```